src/localplayer.cpp

```diff
@@ -176,6 +176,12 @@ void LocalPlayer::move(f32 dtime, Environment *env, f32 pos_max_d,
 	// Maximum distance over border for sneaking
 	f32 sneak_max = BS*0.4;
 
+	// If not touching_ground then we cannot let player sneak
+	if (!touching_ground)
```

Comment:
we do

    if (!touching_ground) {

not 

     if (!touching_ground)
     {

But I guess that can be fixed by someone else, the person that merges.